package.json

```diff
@@ -27,7 +27,8 @@
     "istanbul": "^0.4.1",
     "js-beautify": "^1.5.10",
     "mocha": "^3.4.2",
-    "should": "^11.2.1"
+    "should": "^11.2.1",
```

Comment:
Why we need `should` here?

Reply:
Can you elaborate? The tests use `should`.

Reply:
@Hoten oh, sorry my mistake, all good :+1: